spring/spring-boot-3/remote/src/main/java/org/infinispan/spring/starter/remote/actuator/RemoteInfinispanCacheMeterBinder.java

```diff
@@ -70,8 +72,23 @@ protected void bindImplementationSpecificMetrics(MeterRegistry registry) {
             .description("Time elapsed in seconds since the last statistics reset")
             .register(registry);
 
-      averages(registry);
-      nearCacheMetrics(registry);
+      try {
+         averages(registry);
+         nearCacheMetrics(registry);
+      } catch (HotRodClientException exception) {
+         // Statistics will be all 0L when the connected user is not authorized to read the cache statistic
+         logger.warning(exception.getMessage());
+      }
+   }
+
+   private Long callMetric(Supplier<Long> supplier) {
+      try {
+         return supplier.get();
+      } catch (HotRodClientException exception) {
+         // Statistics will be all 0L when the connected user is not authorized to read the cache statistic
+         logger.warning(exception.getMessage());
```

Comment:
maybe not optimal to log all the cache-names that the client doesn't have access to? Maybe instead log the ones it does have access to

Reply:
I think we need to reflect that an authorization exception has been encountered, at least once, so that it's possible for the user to take some action. Otherwise it's possible that a user will always have stats with value 0 due to an authorization missconfiguration. I agree though that this shouldn't be per stat call.